client/telemetry/TelemetryEventClient.go

```diff
@@ -283,3 +284,53 @@ func (impl *TelemetryEventClientImpl) HeartbeatEventForTelemetry() {
 		Properties: prop,
 	})
 }
+
+
+func (impl *TelemetryEventClientImpl) GetUCID() (*PosthogData, error) {
+	clusterBean, err := impl.clusterService.FindOne(cluster.ClusterName)
```

Comment:
we should write a single method for fetching `UCID`
duplicate code

Reply:
fixed